### PR TITLE
[gas] fix gas formulae for exists_at & table natives (load resource) 

### PR DESCRIPTION
**Pull request by vgao1996**
This fixes two gas issues
1. `exists_at` is not charging for loading resources
2. The table natives double charge the base read fee when an item has already been loaded

Note this is a continuation of https://github.com/aptos-labs/aptos-core/pull/6852

### CODE REVIEW

File: aptos-move/aptos-gas/src/aptos_framework.rs [hunk 1]
```diff
@@ -116,5 +116,8 @@ crate::natives::define_gas_parameters_for_natives!(GasParameters, "aptos_framewo
     [.aggregator.destroy.base, "aggregator.destroy.base", 500 * MUL],
     [.aggregator_factory.new_aggregator.base, "aggregator_factory.new_aggregator.base", 500 * MUL],
 
-    [.object.exists_at.base_cost, { 7.. => "object.exists_at.base" }, 250 * MUL]
+    [.object.exists_at.base_cost, { 7.. => "object.exists_at.base" }, 250 * MUL],
+    // These are dummy value, they copied from storage gas in aptos-core/aptos-vm/src/aptos_vm_impl.rs
```
Comment (msmouse) [inline aptos-move/aptos-gas/src/aptos_framework.rs:120]: nit: ... they *are* copied ...

File: aptos-move/aptos-gas/src/table.rs [hunk 2]
```diff
@@ -5,8 +5,7 @@ use crate::gas_meter::EXECUTION_GAS_MULTIPLIER as MUL;
 use move_table_extension::GasParameters;
 
 crate::natives::define_gas_parameters_for_natives!(GasParameters, "table", [
-    // Note(Gas): These are legacy parameters for loading from storage so they do not
-    //            need to be multiplied.
+    // These are dummy value, they copied from storage gas in aptos-core/aptos-vm/src/aptos_vm_impl.rs
```
Comment (msmouse) [inline aptos-move/aptos-gas/src/table.rs:8]: ditto

**Review by msmouse — APPROVED** [review]

File: aptos-move/aptos-gas/src/aptos_framework.rs (hunk 1 above)
Comment (vgao1996) [reply aptos-move/aptos-gas/src/aptos_framework.rs:120]: @davidiw your grammar instructor Alden has a word with you XD

Comment (davidiw) [reply aptos-move/aptos-gas/src/aptos_framework.rs:120]: I mean, I'm shocked by the quality of copying. Afaik I have nothing to do with this. And the blame agrees with me.

**Review by davidiw — APPROVED** [review]

**Pull request merged**